.github/workflows/build-test.yml

```diff
@@ -34,7 +46,23 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -r requirements.txt
+          python -m pip install -r requirements.txt
+        # Need tags for scm to provide a proper version
```

Comment:
better follow the spaces next line

Reply:
@yihong0618 are you looking for a space before that comment? or where do you want a line space?

Reply:
I meant that this comment is better aligned with the beginning of the next line, sorry I didn't express it properly

Reply:
oh sure! i follow. Great suggestion and no worries I just wasn't quite sure what you were pointing out!! i see how. I'll fix it locally and will push up soon. Great suggestion!